.github/workflows/test.yml

```
@@ -32,3 +32,5 @@ jobs:
           max_threshold: 1000
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+          
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)